### ISSUE TITLE
obtaining the workflow fails

### ISSUE THREAD

**Issue by rguden** [epi2me-labs/wf-alignment#31]
### Operating System

Other Linux (please specify below)

### Other Linux

openSUSE Leap 15.4

### Workflow Version

v.1.1.2

### Workflow Execution

Command line

### CLI command run

nextflow run epi2me-labs/wf-alignment --help

### Workflow Execution - CLI Execution Profile

standard (default)

### What happened?

Dear EPI2ME,

I am trying to obtain the wf-alignment workflow, but I get an error after pulling the repository in to the assets folder of Nextflow.  Could you kindly let me know what is the problem?

```
rguden@django:~/.nextflow/assets/epi2me-labs/wf-alignment> nextflow run epi2me-labs/wf-alignment --help

 N E X T F L O W   ~  version 24.04.1

Launching `https://github.com/epi2me-labs/wf-alignment` [trusting_meucci] DSL2 - revision: e1fd7a51dc [master]

ERROR ~ Script compilation error
- file : /media/HD3/home/rguden/.nextflow/assets/epi2me-labs/wf-alignment/main.nf
- cause: Invalid output definition @ line 411, column 5.
       output(jb2_conf.concat(results))
       ^

1 error


 -- Check '.nextflow.log' file for details
```

I attached the '.nextlow.log' file in the 'Relevant log output' below.

### Relevant log output

```shell
May-26 23:02:04.554 [main] DEBUG nextflow.cli.Launcher - $> nextflow run epi2me-labs/wf-alignment --help
May-26 23:02:04.692 [main] DEBUG nextflow.cli.CmdRun - N E X T F L O W  ~  version 24.04.1
May-26 23:02:04.733 [main] DEBUG nextflow.plugin.PluginsFacade - Setting up plugin manager > mode=prod; embedded=false; plugins-dir=/media/HD3/home/rguden/.nextflow/plugins; core-plugins: nf-amazon@2.5.2,nf-azure@1.6.0,nf-cloudcache@0.4.1,nf-codecommit@0.2.0,nf-console@1.1.3,nf-ga4gh@1.3.0,nf-google@1.13.2,nf-tower@1.9.1,nf-wave@1.4.2
May-26 23:02:04.754 [main] INFO  o.pf4j.DefaultPluginStatusProvider - Enabled plugins: []
May-26 23:02:04.755 [main] INFO  o.pf4j.DefaultPluginStatusProvider - Disabled plugins: []
May-26 23:02:04.761 [main] INFO  org.pf4j.DefaultPluginManager - PF4J version 3.10.0 in 'deployment' mode
May-26 23:02:04.782 [main] INFO  org.pf4j.AbstractPluginManager - No plugins
May-26 23:02:04.806 [main] DEBUG nextflow.scm.ProviderConfig - Using SCM config path: /media/HD3/home/rguden/.nextflow/scm
May-26 23:02:06.112 [main] DEBUG nextflow.scm.AssetManager - Git config: /media/HD3/home/rguden/.nextflow/assets/epi2me-labs/wf-alignment/.git/config; branch: master; remote: origin; url: https://github.com/epi2me-labs/wf-alignment.git
May-26 23:02:06.162 [main] DEBUG nextflow.scm.RepositoryFactory - Found Git repository result: [RepositoryFactory]
May-26 23:02:06.182 [main] DEBUG nextflow.scm.AssetManager - Git config: /media/HD3/home/rguden/.nextflow/assets/epi2me-labs/wf-alignment/.git/config; branch: master; remote: origin; url: https://github.com/epi2me-labs/wf-alignment.git
May-26 23:02:07.088 [main] DEBUG nextflow.config.ConfigBuilder - Found config local: /media/HD3/home/rguden/.nextflow/assets/epi2me-labs/wf-alignment/nextflow.config
May-26 23:02:07.092 [main] DEBUG nextflow.config.ConfigBuilder - Parsing config file: /media/HD3/home/rguden/.nextflow/assets/epi2me-labs/wf-alignment/nextflow.config
May-26 23:02:07.105 [main] DEBUG n.secret.LocalSecretsProvider - Secrets store: /media/HD3/home/rguden/.nextflow/secrets/store.json
May-26 23:02:07.109 [main] DEBUG nextflow.secret.SecretsLoader - Discovered secrets providers: [nextflow.secret.LocalSecretsProvider@15e8f9b2] - activable => nextflow.secret.LocalSecretsProvider@15e8f9b2
May-26 23:02:07.126 [main] DEBUG nextflow.config.ConfigBuilder - Applying config profile: `standard`
May-26 23:02:07.396 [main] DEBUG nextflow.cli.CmdRun - Applied DSL=2 from script declaration
May-26 23:02:07.398 [main] DEBUG nextflow.cli.CmdRun - Launching `https://github.com/epi2me-labs/wf-alignment` [trusting_meucci] DSL2 - revision: e1fd7a51dc [master]
May-26 23:02:07.402 [main] DEBUG nextflow.plugin.PluginsFacade - Plugins default=[]
May-26 23:02:07.402 [main] DEBUG nextflow.plugin.PluginsFacade - Plugins resolved requirement=[]
May-26 23:02:07.488 [main] DEBUG nextflow.Session - Session UUID: 9d16264b-309d-4c6b-8431-5593695b1914
May-26 23:02:07.488 [main] DEBUG nextflow.Session - Run name: trusting_meucci
May-26 23:02:07.489 [main] DEBUG nextflow.Session - Executor pool size: 24
May-26 23:02:07.502 [main] DEBUG nextflow.file.FilePorter - File porter settings maxRetries=3; maxTransfers=50; pollTimeout=null
May-26 23:02:07.510 [main] DEBUG nextflow.util.ThreadPoolBuilder - Creating thread pool 'FileTransfer' minSize=10; maxSize=72; workQueue=LinkedBlockingQueue[10000]; allowCoreThreadTimeout=false
May-26 23:02:07.539 [main] DEBUG nextflow.cli.CmdRun -
  Version: 24.04.1 build 5913
  Created: 20-05-2024 09:48 UTC (11:48 CEST)
  System: Linux 5.14.21-150400.22-default
  Runtime: Groovy 4.0.21 on OpenJDK 64-Bit Server VM 11.0.15+10-suse-150000.3.80.1-x8664
  Encoding: UTF-8 (UTF-8)
  Process: 24240@django.ugent.be [157.193.164.181]
  CPUs: 24 - Mem: 62.8 GB (2.9 GB) - Swap: 65 GB (61.3 GB)
May-26 23:02:07.570 [main] DEBUG nextflow.Session - Work-dir: /media/HD3/home/rguden/.nextflow/assets/epi2me-labs/wf-alignment/work [ext2/ext3]
May-26 23:02:07.627 [main] DEBUG nextflow.executor.ExecutorFactory - Extension executors providers=[]
May-26 23:02:07.643 [main] DEBUG nextflow.Session - Observer factory: DefaultObserverFactory
May-26 23:02:07.735 [main] DEBUG nextflow.cache.CacheFactory - Using Nextflow cache factory: nextflow.cache.DefaultCacheFactory
May-26 23:02:07.756 [main] DEBUG nextflow.util.CustomThreadPool - Creating default thread pool > poolSize: 25; maxThreads: 1000
May-26 23:02:07.852 [main] DEBUG nextflow.Session - Session start
May-26 23:02:07.857 [main] DEBUG nextflow.trace.TraceFileObserver - Workflow started -- trace file: /media/HD3/home/rguden/.nextflow/assets/epi2me-labs/wf-alignment/output/execution/trace.txt
May-26 23:02:07.868 [main] DEBUG nextflow.Session - Using default localLib path: /media/HD3/home/rguden/.nextflow/assets/epi2me-labs/wf-alignment/lib
May-26 23:02:07.873 [main] DEBUG nextflow.Session - Adding to the classpath library: /media/HD3/home/rguden/.nextflow/assets/epi2me-labs/wf-alignment/lib
May-26 23:02:07.874 [main] DEBUG nextflow.Session - Adding to the classpath library: /media/HD3/home/rguden/.nextflow/assets/epi2me-labs/wf-alignment/lib/nfcore_external_java_deps.jar
May-26 23:02:09.191 [main] DEBUG nextflow.script.ScriptRunner - Parsed script files:
May-26 23:02:09.202 [main] ERROR nextflow.cli.Launcher - Script compilation error
- file : /media/HD3/home/rguden/.nextflow/assets/epi2me-labs/wf-alignment/main.nf
- cause: Invalid output definition @ line 411, column 5.
       output(jb2_conf.concat(results))
       ^

1 error

nextflow.exception.ScriptCompilationException: Script compilation error
- file : /media/HD3/home/rguden/.nextflow/assets/epi2me-labs/wf-alignment/main.nf
- cause: Invalid output definition @ line 411, column 5.
       output(jb2_conf.concat(results))
       ^

1 error

        at nextflow.script.ScriptParser.parse0(ScriptParser.groovy:196)
        at nextflow.script.ScriptParser.parse(ScriptParser.groovy:206)
        at nextflow.script.ScriptRunner.parseScript(ScriptRunner.groovy:229)
        at nextflow.script.ScriptRunner.execute(ScriptRunner.groovy:136)
        at nextflow.cli.CmdRun.run(CmdRun.groovy:372)
        at nextflow.cli.Launcher.run(Launcher.groovy:503)
        at nextflow.cli.Launcher.main(Launcher.groovy:657)
Caused by: org.codehaus.groovy.control.MultipleCompilationErrorsException: startup failed:
Script_e48487325d311a2e: 411: Invalid output definition @ line 411, column 5.
       output(jb2_conf.concat(results))
       ^

1 error

        at org.codehaus.groovy.control.ErrorCollector.failIfErrors(ErrorCollector.java:292)
        at org.codehaus.groovy.control.CompilationUnit$IPrimaryClassNodeOperation.doPhaseOperation(CompilationUnit.java:980)
        at org.codehaus.groovy.control.CompilationUnit.processPhaseOperations(CompilationUnit.java:692)
        at org.codehaus.groovy.control.CompilationUnit.compile(CompilationUnit.java:666)
        at groovy.lang.GroovyClassLoader.doParseClass(GroovyClassLoader.java:373)
        at groovy.lang.GroovyClassLoader.lambda$parseClass$2(GroovyClassLoader.java:316)
        at org.codehaus.groovy.runtime.memoize.StampedCommonCache.compute(StampedCommonCache.java:163)
        at org.codehaus.groovy.runtime.memoize.StampedCommonCache.getAndPut(StampedCommonCache.java:154)
        at groovy.lang.GroovyClassLoader.parseClass(GroovyClassLoader.java:314)
        at groovy.lang.GroovyShell.parseClass(GroovyShell.java:572)
        at groovy.lang.GroovyShell.parse(GroovyShell.java:585)
        at groovy.lang.GroovyShell.parse(GroovyShell.java:639)
        at groovy.lang.GroovyShell.parse(GroovyShell.java:643)
        at nextflow.script.ScriptParser.parse0(ScriptParser.groovy:175)
        ... 6 common frames omitted
```

### Were you able to successfully run the latest version of the workflow with the demo data?

no

**Comment by julibeg** [comment]
Hi @rguden, it appears that a new feature introduced in the latest Nextflow release breaks some of our workflows. We are investigating the best way to fix this.

For the time being, please use Nextflow v23.10.1 (by setting the `NXF_VER` environment variable to 23.10.1).

Many thanks for your patience!

**Comment by julibeg** [comment]
Hi @rguden, this has been fixed in the latest release (v1.2.0). Please let us know if you still run into the same issue after updating. Cheers!
